src/go/util/url_util.go

```diff
@@ -142,3 +142,22 @@ func ExtraAddressFromURI(jwksUri string) (string, error) {
 	}
 	return fmt.Sprintf("%s:%v", hostname, port), nil
 }
+
+const (
```

Comment:
shouldn't these in util.go?

Reply:
They are private const and only used by FetchConfigURL/FetchRolloutsURL